### ISSUE TITLE
Inplace Batch Normalization + RELU question

### ISSUE THREAD

**Issue by zeno40** [intel/mkl-dnn#447]
Hi,

I'm trying to make a c++ class called ConvolutionBatchNormReLU to perform convolution + inplace batch normalization with relu activation. I would like to perform the batch normalization inplace to reduce the overal memory load. For some unknown reason it does't behave like it should. I followed the guidelines in the documentation but the network doesn't train. Is there some sample code of doing a convolution + batch-normalization+relu inplace?

thanks

**Comment by shelleygoel** [comment]
Hi zen040,

Convolution w/ batch norm is only supported for inference , not training. But BatchNorm supports inplace ReLU for training.

**Comment by zeno40** [comment]
Thanks for the response. Now I understand.
